data/foxcraft_dungeon_loot/loot_tables/items/rare/ima_leave.json

```diff
@@ -0,0 +1,82 @@
+{
+  "pools": [
+    {
+      "rolls": 1,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:warped_fungus_on_a_stick"
+        }
+      ]
+    }
+  ],
+  "functions": [
+    {
+      "function": "minecraft:set_name",
+      "name": {
+        "text": "Ima Leave",
+        "color": "aqua",
+        "italic": false
+      }
+    },
+    {
+      "function": "minecraft:set_lore",
+      "lore": [
+        "Ight, Ima head out",
+        [
+          "Rarity: ",
+          {
+            "text": "Rare",
+            "color": "aqua",
+            "italic": true
+          }
+        ],
+        "",
+        [
+          {
+            "text": "Make Like A Tree: ",
+            "color": "aqua",
+            "italic": false
+          },
+          {
+            "text": "Instantly teleports",
+            "color": "gray",
+            "italic": false
+          }
+
+        ],
+        [
+          {
+            "text": "you 10,000 meters into the sky",
+            "color": "gray",
+            "italic": false
+          }
+        ],
+        "",
```

Comment:
Multiple perks shouldn't have white space between them, keeps the chance that the lore will just overrun the bottom of the window lower, but good to follow as a convention

Reply:
done